src/Assignments/ProblemIframe.tsx

```diff
@@ -23,14 +26,16 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
     problem,
     setProblemStudentGrade,
     workbookId,
-    readonly = false
+    readonly = false,
 }) => {
     const iframeRef = useRef<HTMLIFrameElement>(null);
     const [renderedHTML, setRenderedHTML] = useState<string>('');
     const [loading, setLoading] = useState(true);
     const [error, setError] = useState('');
     const [height, setHeight] = useState('100vh');
 
+    const { setLastSavedAt, setLastSubmittedAt } = useCurrentProblemState();
```

Comment:
I'm assuming this does break the grades view of an attempt?

Reply:
I have corresponding changes to the backend behavior that hinges on a workbookID being included.